test/source/TestFixedFunction.cpp

```diff
@@ -2,10 +2,11 @@
 // Copyright (c) Electronic Arts Inc. All rights reserved.
 /////////////////////////////////////////////////////////////////////////////
 
+#include <EASTL/fixed_function.h>
```

Comment:
Is this change required?  EABase.h should be fine as the first header.

Reply:
Moved it below `eabase.h` and `eaassert.h`, and threw in a comment to explain why it's before `EASTLTest.h` so the reasoning is clearer for future readers of that file.